.changeset/metal-jars-end.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Let me pin that comment here... 
I seriously can't tell if those failing VRTs are a thing or not 😐 Locally `visual-testing-app` doesn't show the issue with positioning of the bottom border. I also tried using the `design-system` and `data-table` packages in a new Custom App and it also seems to work fine
<img width="621" alt="Screenshot 2022-11-30 at 14 40 21" src="https://user-images.githubusercontent.com/49066275/204815297-a5d65048-39db-418d-9f7e-167e443497d7.png">